src/main/java/edu/uml/diet/model/BasicFood.java

```diff
@@ -160,6 +202,8 @@ public boolean equals(Object o) {
         if (fat != basicFood.fat) return false;
         if (protein != basicFood.protein) return false;
         if (!name.equals(basicFood.name)) return false;
+        if (householdWeight != basicFood.householdWeight) return false;
+        if (!householdWeightDescription.equals(basicFood.householdWeightDescription)) return false;
 
```

Comment:
even one line if statements should have braces. See Java coding conventions as to why http://www.oracle.com/technetwork/java/codeconventions-150003.pdf